lib/core/text/abstract_text.nit

```diff
@@ -1478,6 +1467,11 @@ abstract class Buffer
 	# Letters that follow a letter are lowercased
 	# Letters that follow a non-letter are upcased.
 	#
+	# Uses `self` as `source` by default, `source` can be changed to any
+	# text shorter or as long as `self`.
```

Comment:
I think it does not clearly explain what `source` does. May I suggest something like the following:

> When the `source` is specified, instead of capitalizing `self`, capitalizes `source` and replaces the first characters of `self` by the result. `source` must be at most as long as `self`.